stl/inc/xatomic_wait.h

```diff
@@ -48,8 +48,10 @@ void __stdcall __std_atomic_notify_all_direct(const void* _Storage) noexcept;
 // The "indirect" functions are used when the size is not 1, 2, 4, or 8; these notionally wait on another value which is
 // of one of those sizes whose value changes upon notify, hence "indirect". (As of 2020-07-24, this always uses the
 // fallback SRWLOCK and CONDITION_VARIABLE implementation but that is not contractual.)
-int __stdcall __std_atomic_wait_indirect(
-    const void* _Storage, const void* _Comparand, size_t _Size, unsigned long _Remaining_timeout) noexcept;
+using _Atomic_wait_indirect_callback = bool(__stdcall*)(const void* _Parameter, const void* _Comparand);

```

Comment:
Can/should this function pointer type be marked as `noexcept`? (I'm not sure if this is ever compiled as C++14.) It appears in an `extern "C"` signature so name mangling shouldn't be a concern, and it seems valuable to tell the compiler that it can omit EH enforcement. (Usually we don't worry too much about marking `noexcept` on internal helpers, but virtuals and separately compiled functions are important places to think about since the compiler can't see through them.)

Reply:
It appears to work. I went ahead and committed, tests will tell for sure

Reply:
Tests complain. I prefer skipping this `noexcept` then.